### PR TITLE
docs: Update get-started-next-steps.md

### PR DESCRIPTION
**Pull request by sasakrsmanovic**
GitHub Issue (If applicable): closes #

- Documentation content changes

## PR Type

What kind of change does this PR introduce?

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 281b359</samp>

This change improves the documentation for getting started with Uno Platform by reorganizing and updating the `get-started-next-steps.md` file. It categorizes the learning resources by source and format, and removes some obsolete or unnecessary sections and links.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by jeromelaban — APPROVED** [review]

File: doc/articles/get-started-next-steps.md [hunk 1]
```diff
@@ -2,28 +2,30 @@
...
 
-Christian Findlay developed a great video tutorial to get you started with Uno Platform course on Udemy. The course walks you through a simple UI scenario and shows you how to build a single-source app for iOS, Android, Browser, and macOS.  [Uno Platform course on Udemy]( https://www.udemy.com/course/introduction-to-uno-platform/?referralCode=C9FE308096EADFB5B661)
 
-#### Skye Hoefling – Getting Started with Uno Platform
+#### Pluralsight - Uno Platform Fundamentals (Paid)
+
+If you already have a subscription to Pluralsight, you can find our [Uno Platform Fundamentals course](https://www.pluralsight.com/courses/uno-platform-fundamentals). By the end of the course, you will have created your first Uno Platform solution in Visual Studio and you’ll have the skills and knowledge needed to build multi-platform apps with Uno Platform. 
+
+The course covers how to build a user interface in XAML, how to apply the MVVM pattern, how to adapt your user interface to different screen sizes, how to write platform-specific code and how to connect your application to a data source. _If you’re not Pluralsight subscriber, you can still use the course it with a 30-day trial._
```
Comment (ajpinedam) [inline doc/articles/get-started-next-steps.md:26]:
```suggestion
If you already have a subscription to Pluralsight, you can find our [Uno Platform Fundamentals course](https://www.pluralsight.com/courses/uno-platform-fundamentals). By the end of the course, you will have created your first Uno Platform solution in Visual Studio, and you’ll have the skills and knowledge needed to build multi-platform apps with Uno Platform. 

The course covers how to build a user interface in XAML, how to apply the MVVM pattern, how to adapt your user interface to different screen sizes, how to write platform-specific code and how to connect your application to a data source. _If you’re not a Pluralsight subscriber, you can still use the course with a 30-day trial._
```

File: doc/articles/get-started-next-steps.md [hunk 2]
```diff
@@ -43,84 +45,62 @@ Microsoft MVP Skye Hoefling put together [Uno Platform: Getting Started Series](
...
-#### Microsoft blogs on Uno Platform
 
-- WinUI Team Blog [Building cross-platform apps with WinUI and Uno]( https://blogs.windows.com/windowsdeveloper/2021/03/11/build-cross-platform-applications-with-winui-and-uno-platform/). This tutorial walks you through building a simple same-codebase app which uses WinUI InfoBar control and runs on Windows 10, WebAssembly and Android.
-- Here are two articles that will help developers [Getting started with the Surface Duo]( https://devblogs.microsoft.com/surface-duo/tag/uno-platform/).
 
-#### Building a Todo App with Uno Platform
+#### Building a Todo App with Uno Platform (Free)
 
-[Steven Giesel](https://github.com/linkdotnet). recently explored the in’s and out of Uno Platform and in turn documented his experience with a great blog series that dives further into What is Uno Platform and how to build a multi-platform Todo App.
+[Steven Giesel](https://github.com/linkdotnet). explored the in’s and out of Uno Platform and in turn documented his experience with a great blog series that dives further into What is Uno Platform and how to build a multi-platform Todo App.
```
Comment (ajpinedam) [inline doc/articles/get-started-next-steps.md:53]:
```suggestion
[Steven Giesel](https://github.com/linkdotnet) explored the ins and outs of Uno Platform and in turn, documented his experience with a great blog series that dives further into What is Uno Platform and how to build a multi-platform Todo App.
```

File: doc/articles/get-started-next-steps.md [hunk 3]
```diff
@@ -43,84 +45,62 @@ Microsoft MVP Skye Hoefling put together [Uno Platform: Getting Started Series](
...
+## Microsoft docs and blogs on Uno Platform (Free) 
 
-## Uno Platform Resources
+- WinUI Documentation - Basic (2 posts) [Build a Hello World app using C# and WinUI 3 / Windows App SDK](https://learn.microsoft.com/en-us/windows/apps/how-tos/hello-world-winui3) leads you through setup of a very basic WinUI Application. The second blog post [How to target multiple platforms with your WinUI 3 app](https://learn.microsoft.com/en-us/windows/apps/how-tos/hello-world-winui3) walks you through taking that same Hello World application cross-platform using Uno PLatform.
+- WinUI Documentation - Advanced (2 posts) [Tutorial: Create a simple photo viewer with WinUI 3](https://learn.microsoft.com/en-us/windows/apps/get-started/simple-photo-viewer-winui3?tabs=cs) leads you through creation of a  WinUI-based Photo Viewing Application. The second blog post [Tutorial: Build a simple photo viewer that targets multiple platforms](https://learn.microsoft.com/en-us/windows/apps/get-started/uno-simple-photo-viewer) walks you through taking the Photi Viewing application cross-platform using Uno PLatform.
```
Comment (ajpinedam) [inline doc/articles/get-started-next-steps.md:82]:
```suggestion
- WinUI Documentation - Advanced (2 posts) [Tutorial: Create a simple photo viewer with WinUI 3](https://learn.microsoft.com/en-us/windows/apps/get-started/simple-photo-viewer-winui3?tabs=cs) leads you through creation of a  WinUI-based Photo Viewing Application. The second blog post [Tutorial: Build a simple photo viewer that targets multiple platforms](https://learn.microsoft.com/en-us/windows/apps/get-started/uno-simple-photo-viewer) walks you through taking the Photo Viewing application cross-platform using Uno PLatform.
```

**Review by ajpinedam — APPROVED** [review]
Looks good but added some comments/suggestions

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.0

**Pull request merged**